src/Serilog.AspNetCore/Serilog.AspNetCore.csproj

```diff
@@ -20,6 +20,14 @@
     <RootNamespace>Serilog</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
These can go in the long list of properties already present above; we only tend to use property groups when different conditions need to be applied.